mikeio/dataset/_dataarray.py

```diff
@@ -835,13 +863,11 @@ def sel(
             if isinstance(idx, tuple):
                 # TODO: support for dfs3
                 assert len(idx) == 2
-                t_ax_offset = 1 if self._has_time_axis else 0
                 ii, jj = idx
                 if jj is not None:
-                    da = da.isel(idx=jj, axis=(0 + t_ax_offset))
```

Comment:
why is the axis arg no longer necessary?

Reply:
the info is still necessary, just with utilizing that we have a more intuitive syntax, ie. `.isel(x=idx)` or `.isel(y=idx)` instead of `.isel(idx,axis="x")`